### PR TITLE
CSS Framework: Updated section about overlay and shadow

### PR DESCRIPTION
**Pull request by jaspermdegroot**
Updated the section about overlay and shadow on the CSS Framework page. See PR https://github.com/jquery/jquery-ui/pull/1436

### CODE REVIEW

File: pages/theming/css-framework.html [hunk 1]
```diff
@@ -111,9 +111,9 @@
 <h4>Overlay &amp; Shadow</h4>
 <ul>
 	<li>
-		<code>.ui-widget-overlay</code>: Applies 100% width &amp; height dimensions to an overlay screen, along with background color/texture, and screen opacity.
```
Comment (jzaefferer) [inline pages/theming/css-framework.html]: Isn't that still theme dependent? If you pick or create a theme with an overlay texture, you still get a texture. Assuming we're putting those variables back.

**Comment by jzaefferer** [comment]
Nice, probably would've forgotten about this page, again.

File: pages/theming/css-framework.html (hunk 1 above)
Comment (jaspermdegroot) [inline pages/theming/css-framework.html]: All themes that are used with UI 1.12 use the new default theme.css as template. So the changes in overlay and shadow apply to all themes.

**Comment by scottgonzalez** [comment]
@jaspermdegroot @jzaefferer What's the status of this?

**Comment by jaspermdegroot** [comment]
It turned out that the placeholders in the ```ui-widget-overlay``` rule were still needed (https://github.com/jquery/jquery-ui/commit/d58051ba1a7d06b503853645b13babfc9d8a0242) so it is still possible to have a background texture. I will commit an update and then I think this can be merged in the 1-12 branch.

**Comment by jaspermdegroot** [comment]
@scottgonzalez @jzaefferer - Should be good to go now.

Note: When squashing the two commits "overlay" should be left out of the commit message because there won't be any overlay related changes after all.

**Comment by jzaefferer** [comment]
Could you rebase or replace this PR to get rid of the unrelated commits?

**Comment by jaspermdegroot** [comment]
@jzaefferer - Done.

**Comment by jzaefferer** [comment]
Ah, that's all :-) Looks good.

**Pull request closed without merging**